src/models/EventMapper.php

```diff
@@ -771,12 +771,17 @@ public function createEvent($event, $auto_approve = false)
             'tz_continent',
             'tz_place',
             'contact_name',
-        );
+        ];
+
         $contains_mandatory_fields = !array_diff($mandatory_fields, array_keys($event));
         if (!$contains_mandatory_fields) {
             throw new Exception("Missing mandatory fields");
         }
 
+        if (strtotime($event['start_date']) > strtotime($event['end_date'])) {
```

Comment:
This code is in the wrong place. All other validation checks are in the `EventsController` here: https://github.com/joindin/joindin-api/blob/master/src/controllers/EventsController.php#L202-L206

Also, note that the `start_date` and `end_date` are already unix timestamps, so don't need to be strtotime'd